packages/web/src/pages/asset-detail-page/components/AssetInfoSection.tsx

```diff
@@ -110,8 +110,9 @@ type BannerSectionProps = {
 const BannerSection = ({ mint }: BannerSectionProps) => {
   const { data: coin, isLoading } = useArtistCoin(mint)
   const theme = useTheme()
-  const { ownerId: ownerIdString } = coin ?? {}
-  const ownerId = HashId.parse(ownerIdString)
+  const { ownerId: ownerIdRaw } = coin ?? {}
+  const ownerId =
+    typeof ownerIdRaw === 'string' ? HashId.parse(ownerIdRaw) : ownerIdRaw
```

Comment:
why do we need this? shouldn't it always be string?

Reply:
no, it wasn't and was crashing the asset detail pages. not sure what changed but fixes for now